### PR TITLE
New SplunkPy command !splunk-job-status

### PR DESCRIPTION
**Pull request by moishce**
## Status
- [x] In Progress

## Related Issues
fixes: https://github.com/demisto/etc/issues/22345

## Description
Add a new command to SplunkPy integration for checking the status of a job in Splunk

## Does it break backward compatibility?
   - [ ] Yes
       - Further details:
   - [x] No

### CODE REVIEW

**Comment by ShahafBenYakir** [comment]
@moishce pull from master, solve merge conflicts and push again.

File: Integrations/SplunkPy/SplunkPy.py [hunk 1]
```diff
@@ -546,6 +546,26 @@ def splunk_edit_notable_event_command(proxy):
...
+def splunk_job_status(service):
+
+    job_sid = demisto.args().get('sid')
+    job = service.job(job_sid)
+    status = job.state.content.get('dispatchState')
+    entry_context = {
+        'SID': job_sid,
+        'Status': status
+    }
+    context = {'Splunk.JobStatus(val.SID && val.ID === obj.SID)': entry_context}
```
Comment (ShahafBenYakir) [inline Integrations/SplunkPy/SplunkPy.py]:
```suggestion
    context = {'Splunk.JobStatus(val.SID && val.SID === obj.SID)': entry_context}
```

File: Integrations/SplunkPy/SplunkPy.py [hunk 2]
```diff
@@ -546,6 +546,26 @@ def splunk_edit_notable_event_command(proxy):
...
+    human_readable = tableToMarkdown('Splunk Job Status', entry_context)
+    demisto.results({
+        "Type": entryTypes['note'],
+        "Contents": 'Splunk Job Status',
```
Comment (ShahafBenYakir) [inline Integrations/SplunkPy/SplunkPy.py]:
```suggestion
        "Contents": entry_context,
```

**Comment by kirbles19** [comment]
@moishce Done. Let's regenerate the README

**Pull request closed without merging**